### PR TITLE
Fix Razor location UTs

### PR DESCRIPTION
**Pull request by CristianAmbrosini**
Fix failing UTs on master due to incorrect location on `.razor` files due to in-memory compilation:
- Verify_Razor 
- Verify_RazorExpressions_Locations
- Verify_RazorWithAdditionalLocation

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.TestFramework.Test/TestCases/Dummy.SecondaryLocation_CSharp10.razor
Comment (Tim-Pohlmann) [inline analyzers/tests/SonarAnalyzer.TestFramework.Test/TestCases/Dummy.SecondaryLocation_CSharp10.razor:1]: The file should be called `Dummy.SecondaryLocation.CSharp10.razor` to follow our naming scheme.

File: analyzers/tests/SonarAnalyzer.TestFramework.Test/TestCases/DummyExpressions_CSharp10.razor
Comment (Tim-Pohlmann) [inline analyzers/tests/SonarAnalyzer.TestFramework.Test/TestCases/DummyExpressions_CSharp10.razor:1]: Rename to `DummyExpressions.CSharp10.razor`

File: analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs [hunk 1]
```diff
@@ -159,9 +159,23 @@ public class Sample
     [DataRow("Dummy.SecondaryLocation.cshtml")]
     public void Verify_RazorWithAdditionalLocation(string path) =>
         DummyWithLocation.AddPaths(path)
+            .WithOptions(ParseOptionsHelper.BeforeCSharp10)
             .WithAdditionalFilePath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Product))
             .Verify();
 
+#if NET
```
Comment (Tim-Pohlmann) [inline analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs:166]: Is this required to not fail in the pipeline?

**Review by Tim-Pohlmann — CHANGES_REQUESTED** [review]
The files need renaming. Looks good otherwise.

File: analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs (hunk 1 above)
Comment (CristianAmbrosini) [reply analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs:166]: I don't think so, let's try

**Review by Tim-Pohlmann — APPROVED** [review]
LGTM!

**Pull request merged**